ParisBNF/abb/BNFabb174.xml

```diff
@@ -0,0 +1,342 @@
+                    </msIdentifier>
+                    <msContents>
+                       <summary/>
+
+                             <msItem xml:id="ms_i1">
+                               <locus from="1r" to="47ra">1-89</locus>
+                               <title type="complete" ref="LIT1594Shephe">ዘሄርማ፡ ነቢይ፡</title>
+                               <textLang mainLang="gez"/>
+                                   <incipit xml:lang="gez">
+                                </incipit>
+                                <note>Used for the edition
+                                  <bibl>
+                                     <ptr target="bm:Abbadie1860Hermas"/>
+                                  </bibl>.</note>
+                               </msItem>
+
+                               <msItem xml:id="ms_i1.2">
+                                   <locus from="48ra" to="53ra">91-101</locus>
+                                   <title type="complete" ref="LIT2228Ruleso">Rules of Pachomius</title>
+                                   <textLang mainLang="gez" otherLangs="am"/>
+                                   <incipit xml:lang="gez">
+                                     </incipit>
+                                     <note>
+                                     <bibl>Used for the edition <ptr target="bm:Dillmann1866Chrestomathia"/>
+                                     <citedRange unit="page">57-69</citedRange></bibl>.
+                                     </note>
+                                   </msItem>
+
+                               <msItem xml:id="ms_i1.3">
+                                   <locus from="54ra" to="61ra">103-161</locus>
+                                   <title type="complete" ref="LIT1405GadlaA"></title>
+                                   <textLang mainLang="gez" otherLangs="am"/>
+                                   <incipit xml:lang="gez">
+                                     </incipit>
+                                     <note>Used for the edition
+                                       <bibl><ptr target="bm:ContiRossini1910Abakerazun"/></bibl>.</note>
+                               </msItem>
+
+
+                          </msContents>
+
+                          <physDesc>
+                             <objectDesc form="Codex">
+                                <supportDesc>
+                                   <support>
+                                      <material key="parchment"/>
+                                   </support>
+                                   <extent>
+                                      <measure unit="page">164 (2+162)</measure>
+                                      <measure unit="page" type="blank">12</measure>
+                                        <locus target="#ir">i</locus>,
+                                        <locus target="#25v">48</locus>,
```

Comment:
if page 1 = folio 1, then page 48 is folio 24v, page 90 is folio 45v, folio 102 is 51v, folio 162 is 81v....
all must be calculated correctly, you should divide by 2
if a number divides by 2 (e.g. 90:2 = 45) add  v = 45v; if it does not divide (e.g. 61:2=30,5) then round up and add r = 31r

Reply:
Indeed. There were three frames repeated twice, hence the mistake. I will correct.